ui/src/App.tsx

```diff
@@ -16,7 +16,7 @@ import { darkTheme } from "./Themes/globalThemes";
 
 //Logic
 import { getUrlParams } from "./getUrlParams";
-import { Web3SignerApi } from "./web3signerApi";
+import { Web3SignerApi } from "./logic/web3signerApi";
```

Comment:
wouldnt be a better name for the folder `apis` since everything inside are apis?

Reply:
Done